services/net/notification/NotificationManager.cs

```diff
@@ -394,17 +394,21 @@ private async Task ResetAlertAsync(API.Areas.Services.Models.Content.ContentMode
             if (subscriber.User.AccountType == UserAccountType.Distribution)
             {
                 var users = await this.Api.GetDistributionListAsync(subscriber.UserId);
-                return users.Select(u => new API.Areas.Services.Models.Notification.UserModel());
+                var filteredUsers = users.Where(u => !u.IsVacationMode());
+
+                return filteredUsers.Select(u => new API.Areas.Services.Models.Notification.UserModel(u));
             }
             else
             {
-                return new[] { subscriber.User };
+                return !subscriber.User.IsVacationMode()
```

Comment:
You have already filtered out users that are on vacation mode on line 397.  Why are you doing it again?

Reply:
if `userAccountType` is type of `Distribution` then we will discard those emails with `IsVacationMode`. Otherwise, if `userAccountType` is not a single user, then added if does not have `IsVacationMode`. We want to discard email accounts with IsVacationMode in both conditions. Is this understanding not right?